ivy_tests/test_ivy/helpers/assertions.py

```diff
@@ -149,7 +149,7 @@ def value_test(
             )
 
 
-def check_unsupported_dtype(*, fn, input_dtypes, all_as_kwargs_np):
+def check_unsupported_dtype(*, fn, input_dtypes, all_as_kwargs_np, device="cpu"):
```

Comment:
do we need a default value for this one?

Reply:
Yep seems like we do https://github.com/unifyai/ivy/actions/runs/5465488875/jobs/9950458070?pr=18263#step:3:135